efficientdet/keras/utils_keras_test.py

```diff
@@ -13,27 +13,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-
+from absl import logging
+from absl.testing import parameterized
 import tensorflow as tf
+
 import utils
 from keras import utils_keras
-from absl.testing import parameterized
 
 
 class KerasUtilTest(tf.test.TestCase, parameterized.TestCase):
 
-  @parameterized.parameters(
-      (True),
-      (False),
-  )
-  def test_batch_normalization(self, is_training_bn):
+  @parameterized.named_parameters(
+      ('train_local', True, ''), ('eval_local', False, ''),
+      ('train_tpu', True, 'tpu'), ('eval_tpu', False, 'tpu'))
+  def test_batch_norm(self, is_training, strategy):
     inputs = tf.random.uniform([8, 40, 40, 3])
-    with self.subTest(is_training_bn=is_training_bn):
-      expect_results = utils.batch_norm_act(inputs, is_training_bn, None)
-      tf.keras.backend.set_learning_phase(int(is_training_bn))
```

Comment:
@fsx950223 I believe set_learning_phase is going to be deprecated, so I changed it to directly call bn with is_training.